src/build.d

```diff
@@ -1620,10 +1690,9 @@ auto run(T)(T args)
     return res.output;
 }
 
-version (CRuntime_DigitalMars)
+auto parallel(R)(R range)
```

Comment:
The only purpose of this function was to disable parallelism. Since your now doing that with the job count, we can just remove it.

Reply:
I've kept it as a utility to ensure `foreach(el; range.parallel) { ... }` (e.g. in `checkwhitespace`) uses our own `taskPool` instance instead of `std.parallelism.taskPool`

Reply:
I would just use taskPool directly. Its unexpected that calling .parallel, (normally a standard library function), would be using a global variable from build.d.  The abstraction makes it more complicated than it needs to be in my opinion.

Reply:
I'll change it

Reply:
Done. `TaskPool` and `totalCPUs` are selective imports now to prevent usage of `std.parallelism.parallel`